scripts/issue/31/upgrade_vestedcvx_strat.py

```diff
@@ -0,0 +1,66 @@
+from eth_abi import encode_abi
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+NEW_LOGIC = registry.eth.logic["native.vestedCVX"] 
+DEV_PROXY = registry.eth.badger_wallets.devProxyAdmin
+
+def main(queue="true"):
+    safe = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+    safe.init_badger()
+
+    strat_proxy = safe.badger.strat_bvecvx
+
+    if queue == "true":
+        safe.badger.queue_timelock(
+            target_addr=DEV_PROXY,
+            signature="upgrade(address,address)",
+            data=encode_abi(
+                ["address", "address"],
+                [strat_proxy.address, NEW_LOGIC],
+            ),
+            dump_dir="data/badger/timelock/upgrade_veCVX_strategy/",
+            delay_in_days=2.3,
```

Comment:
2.3 days enough?

i think if we assume posting on thursday, and execing a week later, maybe ~4 days would be safer? this would give signers the whole weekend to review.

Reply:
I set it with a 2.3 ETA because there was a chance that we were going to queue it on Tuesday to exec today. Will change it to 4 days upon loading for it to be executed next week.